wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -75,77 +43,72 @@ export default function ShopModalView(props: ShopModelViewProps) {
-                  />
-                </div>
+                <h1 className="mb-0">{props?.diveShop.orgName}</h1>
               </div>
+            </div>
 
-              {/* <div className="d-flex">
-                {'Added by: '}
-                <a href="#">{props?.diveSite?.newusername}</a>
-              </div> */}
+            <div className="panel border-none">
+              <div className="panel-body">
+                <PlainTextInput
+                  placeHolder={`A little about ${props?.diveShop?.orgName}`}
+                  content={props?.diveShop?.diveShopBio || ''} // diveShopBio doesn't exist yet.
+                  readOnly={!props?.isPartnerAccount}
+                  onSubmit={props?.onDiveShopBioChange}
+                />
+              </div>
             </div>
           </div>
         </div>
         <div className="ml-6">
           <h3 class="text-left">{props?.shopDescription}</h3>
         </div>
       </div>
-      <div className="col-6">
-        <h3>Offered Diving Trips</h3>
-        <div style={{ marginTop: '3%', width: '100%', borderRadius: 15 }}>
-          <div className="container5">
-            {itineraryList
-            && itineraryList.map((itinerary) => {
+      <div className="col-6 panel border-none full-height">
+        <div className="panel-header">
+          <h3>Offered Diving Trips</h3>
+          {props?.isPartnerAccount && (
+            <Button className="mt-2 btn-lg">
+                Add diving event
+            </Button> 
+          )}     
+        </div>
+        <div className={`${style.itineraryList}`}>
+          {props?.itineraryList// in the future, if itineraryList is not empty, render a loading spinner
+            && props?.itineraryList.map((itinerary) => {
               return (
                 <Itinerary
                   key={itinerary.id}
                   itinerary={itinerary}
-                  setSelectedID={setSelectedID}
-                  selectedID={selectedID}
-                  setShopModal={setShopModal}
+                  setSelectedID={props?.setSelectedID}
+                  selectedID={props?.selectedID}
+                  setShopModal={props?.setShopModal}
                 />
               );
             })}
-            {itineraryList.length === 0 && (
-              <div>
-                <p className="noSightings">
-                  No Trips are currently being offered.
-                </p>
-              </div>
-            )}
-          </div>
+          {props?.itineraryList.length === 0 && (
```

Comment:
Ts is complaining here you'll need another "?" after itineraryList to satisfy Ts